docs/writing-rules/pattern-syntax.mdx

```diff
@@ -515,7 +515,27 @@ When referencing an ellipsis metavariable in a rule message or [metavariable-pat
 - message: Call to foo($...ARGS)
 ```
 
-### Displaying matched metavariables in rule messages
+### Anonymous metavariables
+
+Anonymous metavariables are used to specify that there a metavariable exists in the pattern you want to capture.
+
+An anonymous metavariable always takes the form `$_`. Variables such as `$_1` or `$_2` are **not** anonymous. You can use more than one anonymous metavariable in a rule definition.
+
+For example, if you want to specify that a function should **always** have 3 arguments, then you can use anonymous metavariables:
```

Comment:
could we stress why this is different than `function($A, $B, $C)`? id like if we explicitly said that anonymous metavars _do not produce bindings_, and that is why there is no unification that occurs between them

Reply:
Here is the addition:
![image](https://github.com/semgrep/semgrep-docs/assets/20766840/b71fa58c-bd11-4c64-917d-4b489db0f275)